web/base_viewer.js

```diff
@@ -497,8 +500,8 @@ class BaseViewer {
 
     // Fetch a single page so we can get a viewport that will be the default
     // viewport for all pages
-    firstPagePromise
-      .then(firstPdfPage => {
+    Promise.all([firstPagePromise, hasJSActionsPromise])
```

Comment:
This section is pretty important to display things as fast as possible. Could we pass in the hasJSActionsPromise into the page view instead so it's resolved a a later time and we don't have to wait for it to starting showing pages?